zavod/zavod/helpers/text.py

```diff
@@ -107,3 +107,54 @@ def remove_bracketed(text: Optional[str]) -> Optional[str]:
     if text is None:
         return None
     return BRACKETED.sub(" ", text)
+
+def clean_br_cpf(cpf: str) -> str:
+    """Remove punctuation from a CPF number.
+    If it is already clean, it will return it as is.
+
+    The CPF number is a Brazilian tax identification number for individuals
+    that is formatted with punctuation (XXX.XXX.XXX-XX) to make it easier to
+    read. However, when saving the CPF number in the database, it's common
+    to remove the punctuation.
+
+    Args:
+        cpf: The CPF number to be cleaned.
+
+    Returns:
+        The cleaned CPF number.
+    """
+
+    # Remove formatting characters
+    cpf = cpf.replace(".", "").replace("-", "")
+
+    # If the CPF is not 11 digits long, it's invalid
+    if len(cpf) != 11:
+        return ""
```

Comment:
instead of losing data, let's return what we have, broken though it may be.

Reply:
this is important because a) for an id, we don't want invalid numbers to end up giving different entities the same id, and b) a human might still make sense of an invalid ID in an investigative setting.

Reply:
That makes sense, I'll fix that